packages/wonder-blocks-tokens/src/theme/semantic/semantic-color.ts

```diff
@@ -283,59 +279,59 @@ export const semanticColor = {
                     background: core.background.instructive.default,
-                    foreground: core.foreground.inverse.strong,
+                    foreground: core.foreground.knockout.default,
                 },
                 hover: {
                     border: core.border.instructive.default,
                     background: core.background.instructive.default,
-                    foreground: core.foreground.inverse.strong,
+                    foreground: core.foreground.knockout.default,
                 },
                 press: {
                     border: core.border.instructive.strong,
                     background: core.background.instructive.strong,
-                    foreground: core.foreground.inverse.strong,
+                    foreground: core.foreground.knockout.default,
                 },
             },
             destructive: {
                 default: {
                     border: core.transparent,
                     background: core.background.critical.default,
-                    foreground: core.foreground.inverse.strong,
+                    foreground: core.foreground.knockout.default,
                 },
                 hover: {
                     border: core.border.critical.default,
                     background: core.background.critical.default,
-                    foreground: core.foreground.inverse.strong,
+                    foreground: core.foreground.knockout.default,
                 },
                 press: {
                     border: core.border.critical.strong,
                     background: core.background.critical.strong,
-                    foreground: core.foreground.inverse.strong,
+                    foreground: core.foreground.knockout.default,
                 },
             },
 
             neutral: {
                 default: {
                     border: core.transparent,
                     background: core.background.neutral.default,
-                    foreground: core.foreground.inverse.strong,
+                    foreground: core.foreground.knockout.default,
                 },
                 hover: {
                     border: core.border.neutral.default,
                     background: core.background.neutral.default,
-                    foreground: core.foreground.inverse.strong,
+                    foreground: core.foreground.knockout.default,
                 },
                 press: {
                     border: core.border.neutral.strong,
                     background: core.background.neutral.strong,
-                    foreground: core.foreground.inverse.strong,
+                    foreground: core.foreground.knockout.default,
                 },
             },
 
             disabled: {
                 border: core.border.disabled.strong,
                 background: core.border.disabled.strong,
-                foreground: core.foreground.inverse.default,
+                foreground: core.foreground.knockout.default,
```

Comment:
`core.foreground.inverse.default` was `offWhite`, `core.foreground.knockout.default`is `white`. This is a slight change since we don't include `offWhite` in any core foreground colors. Since it's a slight change, it doesn't show in the diffs in Chromatic!

<img width="1724" height="956" alt="Image" src="https://github.com/user-attachments/assets/92ddea2a-a15e-485a-be73-3d25c7f435da" />

Note: TB is not impacted by this

<img width="1725" height="921" alt="Image" src="https://github.com/user-attachments/assets/bb38e3f4-ffee-4027-ba3a-9cb8400572a1" />